JSDemos/Demos/Tabs/Overview/React/App.tsx

```diff
@@ -32,9 +32,26 @@ const App = () => {
   const [showNavigation, setShowNavigation] = React.useState(false);
   const [scrollContent, setScrollContent] = React.useState(false);
   const [fullWidth, setFullWidth] = React.useState(false);
+  const [width, setWidth] = React.useState('auto');
   const [rtlEnabled, setRtlEnabled] = React.useState(false);
+  const [selectedIndex1, setSelectedIndex1] = React.useState(0);
```

Comment:
I suggest we may use array or object approach here as follows
```
const [selectedIndexes, setSelectedIndexes] = React.useState([0, 0, 0]);
```
or 
```
const [selectedIndices, setSelectedIndices] = React.useState({
  indexWithText: 0,
  indexWithIconAndText: 0,
  indexWithIcon: 0,
});
```

It might help in managing these related states more efficiently when we have deal with many similar states

Reply:
Using of setSelectedIndices is more difficult.

Reply:
These methods was removed in [5b9bd3875eddb26acdca83ce94cfe7a767786467].